### DISCUSSION TITLE
October 29th Check-In

### DISCUSSION THREAD

**Discussion by DylanTh20** [ecn310/course-project-taxes-tariffs#15] [General]
How yall feeling

**Comment by umarbilgrammi** [comment]
### **10/29/2024**
- Something someone did to help the team this week: Dylan and Kathryn created the to do logs for the most pressing issues 
- Something I could have done better for the team this week: Help organize and figure out how to fix our data issues 
- An area/suggestion for team improvement: I think we can be more specific with referencing commits when we update our logs

**Comment by Ksarrge** [comment]
Something someone did to help the team this week: Umar did the do file for the data
Something I could have done better for the team this week: Show up to our meeting on time
An area/suggestion for team improvement: team is doing well, maybe discuss which assignments everyone needs to do more specifically

**Comment by DylanTh20** [comment]
Something someone did to help the team this week: Umar and Kathryn worked on the data and fixed our paper summaries
Something I could have done better for the team this week: Helped more with transforming our data 
An area/suggestion for team improvement: I think we can meet for specific issues. Example: purely just to fix the data
